Apps/WebClient/src/ClientApp/src/components/modal/NewDependentComponent.vue

```diff
@@ -250,12 +252,22 @@ export default class NewDependentComponent extends Vue {
                                     />
                                     <b-form-invalid-feedback
                                         :state="
-                                            isValid($v.dependent.dateOfBirth)
+                                            $v.dependent.dateOfBirth.minValue
                                         "
                                     >
                                         Dependent must be under the age of
                                         {{ webClientConfig.maxDependentAge }}
                                     </b-form-invalid-feedback>
+                                    <b-form-invalid-feedback
+                                        :state="
+                                            $v.dependent.dateOfBirth.$dirty
+                                                ? $v.dependent.dateOfBirth
+                                                      .maxValue
+                                                : undefined
+                                        "
+                                    >
+                                        Invalid date
+                                    </b-form-invalid-feedback>
```

Comment:
If the `required` or `minLength` validations fail, there still needs to be feedback displayed for them.  I think they can be covered here under the designation of "Invalid date":

```
$v.dependent.dateOfBirth.$dirty
      ? $v.dependent.dateOfBirth.required || $v.dependent.dateOfBirth.minLength || $v.dependent.dateOfBirth.maxValue
      : undefined
```

Reply:
Fixed